src/main/java/me/xmrvizzy/skyblocker/skyblock/item/ItemRenaming.java

```diff
@@ -0,0 +1,66 @@
+package me.xmrvizzy.skyblocker.skyblock.item;
+
+import java.util.Map;
+
+import com.mojang.brigadier.Command;
+import com.mojang.brigadier.CommandDispatcher;
+
+import me.xmrvizzy.skyblocker.config.SkyblockerConfig;
+import me.xmrvizzy.skyblocker.utils.Utils;
+import net.fabricmc.fabric.api.client.command.v2.ClientCommandManager;
+import net.fabricmc.fabric.api.client.command.v2.ClientCommandRegistrationCallback;
+import net.fabricmc.fabric.api.client.command.v2.FabricClientCommandSource;
+import net.minecraft.client.MinecraftClient;
+import net.minecraft.command.CommandRegistryAccess;
+import net.minecraft.command.argument.TextArgumentType;
+import net.minecraft.item.ItemStack;
+import net.minecraft.nbt.NbtCompound;
+import net.minecraft.text.Text;
+
+public class ItemRenaming {
```

Comment:
Imo `ItemRename` or `RenameItem` sounds better.

Reply:
I've chosen the name `CustomItemNames` for the item renaming, and for the custom dye colours I've chosen `CustomArmorDyeColor`

If you have any better ideas feel free to suggest them, maybe later on these things could be moved into a custom "decoration" package or something to make the purpose clearer.